docs/installation.rst

```diff
@@ -90,7 +90,15 @@ Install Molecule:
 
 .. code-block:: bash
 
-    $ python3 -m pip install --user "molecule[lint]"
+    $ python3 -m pip install --user "molecule[lint] molecule-docker"
+
+Molecule uses docker by default, which is maintained separately as
```

Comment:
That is no longer true: molecule does not use docker by default in any way. The only driver in core is delegated.

The command line you suggested is correct but also similar to `molecule[docker,lint]`, which is shorter.

Reply:
Thanks for the insight. I will edit it

Reply:
This should now be factual. I have also deleted the link the the Configuration page as I don't think it hold enough relevant information regarding the topic (for now)